src/Workspaces/CSharp/Portable/CodeStyle/TypeStyle/TypeStyleHelper.cs

```diff
@@ -306,7 +304,7 @@ public static bool IsPredefinedType(TypeSyntax type)
             }
```

Comment:
this whole helper can get removed in the future.  features should just ask "type.GenerateTypeSyntax()" and they should get the right behavior.